carma_novatel_driver_wrapper/package.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<!--  
+ Copyright (C) 2021 LEIDOS.
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+-->
+<package format="3">
+  <name>carma_novatel_driver_wrapper</name>
+  <version>0.1.0</version>
+  <description>Package containing lifecycle controllers for each carma-platform subsystem</description>
+  <author email="Anish.Deva@leidos.com">Anish Deva</author>
+  <maintainer email="carma@dot.gov">CARMA</maintainer>
+  <license>Apache-2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <build_depend>carma_cmake_common</build_depend>
+  <build_depend>rcpputils</build_depend>
+  <build_depend>ament_cmake_auto</build_depend>
+
+  <depend>carma_ros2_utils</depend>
+  <depend>carma_msgs</depend>
+  <depend>rclcpp</depend>
+  <depend>sensor_msgs</depend>
+  <depend>gps_msgs</depend>
+  <depend>carma_driver_msgs</depend>
+  <depend>novatel_oem7_msgs</depend>
+  <depend>novatel_oem7_driver</depend>
+  <depend>uncertainty_tools</depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>carma_msgs</test_depend>
+  <test_depend>rclcpp</test_depend>
+  <test_depend>gps_msgs</test_depend>
```

Comment:
If something is used in multiple places the depend tag is sufficient. It doesn't need to be duplicated. 
Rule of thumb, if its a dependency for more than one case (build, test, exec) just use depend
If it applies to only one case then use the specific tag (test_depend, build_depend, exec_depend)